lib/ui/settings/advanced_settings_screen.dart

```diff
@@ -119,6 +121,32 @@ class _AdvancedSettingsScreenState extends State<AdvancedSettingsScreen> {
                             const SizedBox(
                               height: 24,
                             ),
+                            MenuItemWidget(
+                              captionedTextWidget: CaptionedTextWidget(
+                                title: S.of(context).hideSharedItems,
+                              ),
+                              menuItemColor: colorScheme.fillFaint,
+                              singleBorderRadius: 8,
+                              alignCaptionedTextToLeft: true,
+                              trailingWidget: ToggleSwitchWidget(
+                                value: () =>
+                                  LocalSettings.instance.getHideSharedItem(),
```

Comment:
Can improve readability if this is changed to a getter function with name `hideSharedItems`